src/main/kotlin/agallipo/Journeys.kt

```diff
@@ -0,0 +1,31 @@
+package agallipo
+
+typealias journey = List<BoardingCard>
+typealias mutableJourney = MutableList<BoardingCard>
+
+open class Journeys {
```

Comment:
Here Journeys is like a util class, that only provides methods. All of them takes a journeysReference that is a mutable list. That lets you to think that you want to keep the state of that list somewhere. That is what objects offers to you: state (properties) + behavior (methods). That said, I think your class should have a property where to store your MutableList<MutableList<BoardingCard>>, so that your class methods can operate over it, without passing it as a parameter and return the resulting list.

then, in BoardingCardSorter.sortTickets() you can create a new instance of Journeys and call directly to its methods
ex: `myJourneys.addJourney(boardingCard)`